#! /usr/bin/env python

PACKAGE='tuw_object_converter'
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("fov_h", double_t, 0, "horizontal field of view in deg", 60, 0, 360)
gen.add("fov_min_dist", double_t, 0, "minimal distance for detection in x", 0, -200, 200)
gen.add("fov_max_dist", double_t, 0, "maximal distance for detection in x", 5, -200, 200)
gen.add("fov_filter", bool_t, 0, "position noise in y", True)

exit(gen.generate(PACKAGE, "detection_to_detection_node", "detection_to_detection"))
